firebase_analytics/src/firebase_analytics.cpp

```diff
@@ -27,9 +29,11 @@ void FirebaseAnalytics_Safe_LogEvent(lua_State*, const char* name, const firebas
 {
 	LogEvent(name, params, number_of_parameters);
 }
+#define THREAD_ATTACHER() dmAndroid::ThreadAttacher threadAttacher;
+#else
+#define THREAD_ATTACHER()
```

Comment:
This define helps to avoid multiple `#if defined(DM_PLATFORM_ANDROID)` checks and makes code clear